src/components/NetworkSettings.vue

```diff
@@ -56,7 +56,7 @@
                             <div class="kiwi-networksettings-section kiwi-networksettings-advanced">
                                 <template v-if="show_advanced">
                                     <label><span>{{$t('settings_encoding')}}: </span> <input v-model="network.connection.encoding" /></label><br />
-                                    <label><span>{{$t('settings_show_raw')}}: </span> <input v-model="settingShowRaw" type="checkbox" /></label><br />
+                                    <label><span class="kiwi-appsettings-showraw-label">{{$t('settings_show_raw')}}: </span> <input v-model="settingShowRaw" type="checkbox" /></label><br />
```

Comment:
Why is this `kiwi-appsettings-showraw-label` class here in the NetworkSettings component, did you mean `kiwi-networksettings-showraw-label`?